### PR TITLE
chore(forms): JSON Schema examples

### PR DESCRIPTION
**Pull request by henit**
- More explanation on the Extended Features page in Forms extension
- A separate page for JSON Schema validation with general information and several examples

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/extended-features/Form/schema-validation.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: 'Schema validation'
+description: '`Form.MainHeading` is a standardized main heading for sections, ensuring default layout, spacing etc.'
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/extended-features/Form/schema-validation.mdx:3]:
```suggestion
description: 'Schema validation can be done with a JSON Schema which makes it possible to describe the data structure and validation needs, both for the individual value, and more complex rules across the data set.'
```

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/extended-features/Form/schema-validation/info.mdx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+const schema = {
+  type: 'object',
+  properties: {
+    name: { type: 'string', minLength: 3 },
+    email: { type: 'string' },
+    birthyear: { type: 'number', minimum: 1900, maximum: 2023 }
+  },
+  required: ['name', 'email', 'birthyear'],
+}
+<Form.Handler data={user}>
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/extended-features/Form/schema-validation/info.mdx:69]:
```suggestion
<Form.Handler data={user} schema={schema}>
```

The `user` is not defined, but that's just hypothetical, right?

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/extended-features/Form/schema-validation/info.mdx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+</Form.Handler>
+```
+
+This makes it possible to create a uniform, testable description and requirements specification for the data, which can be tested independently of frontend code, and used across systems, e.g. frontend and backend.
+
+### Complex schemas
+
+In addition to basic validation as in the example above, JSON Schema can be used for more complex. Examples of definitions supported by the standard are:
+
+- Requirement that the object must not have other properties than those defined in `properties`
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/extended-features/Form/schema-validation/info.mdx:82]: A dot on the ending of all bullet points would be great 🙏

**Review by tujoworker — APPROVED** [review]
Great work 💪 

I had just a few corrections/comments 🙏

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/extended-features.mdx [hunk 4]
```diff
@@ -12,19 +12,33 @@ import ListAdvancedAPIs from './extended-features/ListAdvancedAPIs'
...
+    <Field.Email path="/email" />
+    <Value.Date path="/date" />
     <Form.SubmitButton />
   </Form.Handler>,
 )
 ```
 
+### Schema validation
+
+`Forms.Handler` suports [JSON Schema](https://json-schema.org/) validation. JSON Schema is a flexible standard that makes it possible to describe the data's structure and validation needs, both for the individual value, and more complex rules across the data set. Descriptions and examples of such validation can be found [on a separate page](/uilib/extensions/forms/extended-features/Form/schema-validation).
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/extended-features.mdx:36]:
```suggestion
`Forms.Handler` supports [JSON Schema](https://json-schema.org/) validation. JSON Schema is a flexible standard that makes it possible to describe the data's structure and validation needs, both for the individual value, and more complex rules across the data set. Descriptions and examples of such validation can be found [on a separate page](/uilib/extensions/forms/extended-features/Form/schema-validation).
```

**Review by langz — APPROVED** [review]

**Pull request merged**